test/e2e/test_cluster_configs/network_policy/cilium.json

```diff
@@ -10,20 +11,19 @@
 			"orchestratorProfile": {
 				"orchestratorType": "Kubernetes",
 				"kubernetesConfig": {
-					"networkPolicy": "cilium",
-					"networkPlugin": "cilium"
+					"networkPolicy": "cilium"
 				}
 			},
 			"masterProfile": {
 				"count": 1,
 				"dnsPrefix": "",
-				"vmSize": "Standard_D2_v3"
+				"vmSize": "Standard_D8_v3"
```

Comment:
Why the `vmSize` change?

Reply:
Seeing test flakes, this change didn't move the needle, will revert back. The `GINKGO_FOCUS` addition has proven to be a valid flake reducer.